app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx

```diff
@@ -72,10 +72,7 @@ export const BlockDetailsCard: React.FC<BlockDetailsCardProps> = ({
           <DetailDataRowWrapper>
             <li>
               <DetailDataLabel>{t('block-height')}</DetailDataLabel>
-              <DetailDataValue
-                data-testid="block-height"
-                data-cy="block-height"
-                isLargeText>
+              <DetailDataValue data-testid="block-height" isLargeText>
```

Comment:
Why are we removing the block height here?

Reply:
Removing this should break the test

Reply:
This was a mixup when merging branches. Fixing this.